packages/react-inbox/src/components/Actions/styled.ts

```diff
@@ -11,21 +11,28 @@ export const Action = styled.a<{ href: string; target: string }>(({ theme }) =>
       cursor: "pointer",
       border: "none",
       fontSize: 12,
-      color: theme?.brand?.primaryColor ?? "#9121C2",
+      color: theme?.brand?.inapp?.colors?.invertButtons
```

Comment:
do we need to default to anything? is ther any point after which we are 100% certain of things existing, to avoid `?`?

Reply:
we will for all legacy brands that don't have any colors set

Reply:
added this: https://github.com/trycourier/backend/pull/1500